task/tkn-bundle-oci-ta/0.1/recipe.yaml

```diff
@@ -0,0 +1,9 @@
+---
+base: ../../tkn-bundle/0.1/tkn-bundle.yaml
+add:
+  - use-source
+removeWorkspaces:
+  - source
+preferStepTemplate: true
+replacements:
+  workspaces.source.path: /var/workdir/source
```

Comment:
Should this be `/var/workdir`?

Reply:
I don't think so, the restored source is in [`/var/workdir/source`](https://github.com/konflux-ci/build-definitions/pull/1596/files#diff-d36a1c9b71c7827d73359c092cf21a3169f0bb7a876a4454272132fcbc140e32R60), then the Task reads [from there (example)](https://github.com/konflux-ci/build-definitions/pull/1596/files#diff-d36a1c9b71c7827d73359c092cf21a3169f0bb7a876a4454272132fcbc140e32R90-R95)